uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-assignment.js

```diff
@@ -147,7 +153,8 @@ define([
                     return;
                 }
 
-                $scope.activitySet = assignmentType.definition.activitySets[0];
+                $scope.activity.selected = assignmentType.definition.activitySets[0];
```

Comment:
I am not sure I understand the code. The `$scope.activitySet` is defined as {} at the start and `$scope.activity.selected` as []. Then you assign something to `$scope.activity.selected` and then back to `$scope.activitySet`? There must be an explanation why it works and probably there could be a better potential implementation. Also I am not sure it is a good idea to ng-repeat through {} at all.